cmd/wallet-web/src/pages/DIDConnect.vue

```diff
@@ -5,78 +5,95 @@
+  <div v-else class="flex justify-center w-screen">
+    <div>
+      <Header />
 
-          <md-card-content v-if="userCredentials.length" class="viewport">
-            Here are the credentials being sent to your wallet,
+      <div class="h-auto bg-gray-light rounded-b border border-neutrals-black chapi-container">
+        <div v-if="errors.length">
+          <b>Failed with following error(s):</b>
+          <ul>
+            <li v-for="error in errors" :key="error" class="text-primary-valencia">{{ error }}</li>
+          </ul>
+        </div>
+        <div class="p-5 text-neutrals-dark font-sm">
+          {{ issuerName }} wants to connect to your wallet.
+        </div>
 
-            <md-list class="md-double-line">
-              <md-list-item v-for="credential in userCredentials" :key="credential">
-                <md-icon class="md-primary md-size-2x">perm_identity</md-icon>
-
-                <div class="md-list-item-text">
-                  <span>{{
-                    credential.name ? credential.name : 'Credential name not provided'
-                  }}</span>
-                  <span>{{ credential.description }}</span>
-                </div>
-              </md-list-item>
-            </md-list>
-          </md-card-content>
+        <hr class="mx-5 border border-neutrals-thistle" />
+        <div class="py-6 px-5">
+          <div class="flex z-10 flex-row justify-start items-center p-5 w-full">
+            <div class="flex-none w-12 h-12 border-opacity-10">
+              <!-- todo issue-1055 Read meta data from external urls -->
+              <img src="@/assets/img/generic-issuer-icon.svg" />
+            </div>
+            <div class="flex flex-col">
+              <span
+                class="flex-1 pl-4 font-bold text-left text-neutrals-dark overflow-ellipsis font-sm"
+              >
+                {{ issuerName }}
+              </span>
+              <div class="flex flex-row justify-center items-center pl-4">
+                <img src="@/assets/img/small-lock-icon.svg" />
+                <span class="flex-1 pl-1 text-left text-neutrals-medium overflow-ellipsis font-xs">
+                  {{ requestOrigin }}
+                </span>
+              </div>
+            </div>
+          </div>
+          <!-- Share icon -->
+          <div class="flex flex-col pl-6 w-14 h-14 border-opacity-10">
+            <img src="@/assets/img/share.svg" />
+          </div>
 
-          <md-divider></md-divider>
+          <!-- Wallet to connect-->
+          <div class="flex z-10 flex-row justify-start items-center px-5 w-full">
+            <div class="flex-none w-12 h-12 border-opacity-10">
+              <img :src="walletIcon" />
+            </div>
+            <div class="flex flex-col">
+              <span
+                class="flex-1 pl-4 font-bold text-left text-neutrals-dark overflow-ellipsis font-sm"
+              >
+                <!-- todo still have to finalize the text of the title-->
```

Comment:
Can we stick so some consistent format for TODO's?
`TODO: message` maybe
It'd be easier to look up all TODOs via search afterwards

Reply:
I will address this comment @birtony